### ISSUE TITLE
OCM only mines when I use integrated graphics 

### ISSUE THREAD

**Issue by craticar** [vertcoin-project/one-click-miner-vnext#222]
It only works when I use the onboard graphics card option, it fails other wise. The exact error is "Checks failed. One or more of your miner binaries are showing rapid failures(immediately stop after starting). Please enable debugging under the settings tab and then Save & Restart. Use the debug.log to learn more about what might be going on". I already did that the warning said and googled all I could about the error but there was no solution. 
 Behavior? I expected it to work normally.

 Start OCM-> Checks failed. Is all that happens.

 Website: Github Version release: vertcoin-ocm-2.0-rc17-windows-x64

 Windows 10/Nvidia and Intel

 https://pastebin.com/aF9fSAc5
![chek1-02-27 035802](https://user-images.githubusercontent.com/79741887/109381875-42323d80-78b3-11eb-9978-d1fc4650216a.png)

**Comment by KforG** [comment]
Your GPU does not live up to the minimum requirements 
`[DEBUG] GPU [NVIDIA GeForce GT 710] matched regex [.*NVIDIA.*]`

You need a GPU with 2 Gb Vram or more.

**Comment by craticar** [comment]
> Your GPU does not live up to the minimum requirements
> `[DEBUG] GPU [NVIDIA GeForce GT 710] matched regex [.*NVIDIA.*]`
> 
> You need a GPU with 2 Gb Vram or more.

I know the GPU doesn't meet the minimum requirements but there are others that mine with low Vram nevertheless and the OCM mines fine when I turn on the option to mine with onboard graphics, and if onboard means my integrated card then it is Intel HD 630 which couldn't even run my high graphics games well but, so it doesn't make sense to me.

**Comment by KforG** [comment]
The Intel HD 630 has access to the ram in your computer if you have at least 2 Gb of ram free then the IGPU can mine just fine. Something performing badly in games doesn't equal it can't mine. 

What other GPUs that don't meet the minimum requirements are able to mine?

**Comment by KforG** [comment]
Could you confirm how much Vram your GPU has?

**Comment by craticar** [comment]
Yes, I'm all aware of the shared GPU memory, which is 4GB and the Nvidia is only 1GB but I don't understand how the other that I've seen is the AMD Radeon HD 7750 worked but it gives similar potato speeds as my integrated card. Either they did something to the miner version they had or OCM is a very weird build. Easy Miner and Nicehash didn't give this much problem, nevertheless it's almost impossible to find a decent low profile GPU that fits compact PCs. So, I'll just find a workaround to this.

**Comment by KforG** [comment]
If it was on earlier versions, rc-14 and down, integrated graphics wasn't disabled by default. There also exist 2 Gb models of the HD7750.

If you find a solution, let us know.
